rosbridge_server/scripts/rosbridge_websocket_client.py

```diff
@@ -34,10 +36,12 @@ class WebsocketClientTornado():
     keepalive = None
 
     def __init__(self, uri):
+        self.authenticated = False
```

Comment:
For the next pass: we need to re-do the authentication state logic. This will only authenticate the first user and then think that everyone is authenticated from then on (because there is a single instance of this WebSocket handler object in the websocket_proxy_client version)